ui/packages/catalog-ui-search/src/main/webapp/component/input/number/input-number.view.js

```diff
@@ -28,9 +28,6 @@ module.exports = InputView.extend({
             }
         },
         isValid: function(){
-            if (!this.model.showValidationIssues()){
```

Comment:
❗️ No we want this. Ideally, we would want this in _all_ the input views. Property.js supports configuring inputs as [showValidationIssues: false](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/property/property.js#L37) in which case we should _always_ show the input as valid regardless of whether or not it actually is

What we should be doing instead is making sure that the underlying `Property.js` model here has `showValidationIssues: true` so that this check fails